[2015-12-06 12:31:46] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7016
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9573): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2979): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9573): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9563): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:31:46] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7016
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:40:31] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:40:31] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:40:50] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:40:50] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:43:15] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:43:15] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:43:31] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:43:31] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:43:37] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:43:37] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:44:49] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:44:49] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:46:39] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:46:39] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:48:41] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:48:41] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-06 12:49:38] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#17 {main}  
[2015-12-06 12:49:38] local.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php:7020
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1284): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1329): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1313): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1299): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1237): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(1775): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2256): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\oslonews\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2015-12-29 07:29:47] local.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in C:\xampp\htdocs\oslonews\app\Category.php:52
Stack trace:
#0 C:\xampp\htdocs\oslonews\app\Category.php(52): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 52, Array)
#1 C:\xampp\htdocs\oslonews\app\Providers\MenuServiceProvider.php(47): App\Category::getTopMenuHubCategories('international-e...')
#2 [internal function]: App\Providers\MenuServiceProvider->App\Providers\{closure}(Object(Illuminate\View\View))
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9749): call_user_func_array(Object(Closure), Array)
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14100): Illuminate\Events\Dispatcher->fire('composing: topm...', Array)
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14482): Illuminate\View\Factory->callComposer(Object(Illuminate\View\View))
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#7 C:\xampp\htdocs\oslonews\storage\framework\views\0b9c86839e8965eda89e9ef2e6e2fd10(159): Illuminate\View\View->render()
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14626): include('C:\\xampp\\htdocs...')
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#10 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#12 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#13 C:\xampp\htdocs\oslonews\storage\framework\views\9698c9b52c283cddfbf00a1591f743da(43): Illuminate\View\View->render()
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14626): include('C:\\xampp\\htdocs...')
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#16 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#17 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#18 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#19 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15623): Illuminate\View\View->render()
#20 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15138): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#21 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7688): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#22 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8915): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#23 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8916): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#28 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8901): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\PagesController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'index')
#29 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7855): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'index')
#30 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7826): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7479): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#36 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7480): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#37 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7468): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7453): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2302): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#40 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#41 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#42 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2926): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#43 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#44 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#45 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12949): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#46 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#47 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#48 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11536): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#49 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#50 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#51 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12686): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#52 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#53 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#54 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12623): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#55 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#56 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#57 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#58 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#59 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#60 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#61 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#62 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#63 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#64 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#65 {main}

Next exception 'ErrorException' with message 'Trying to get property of non-object (View: C:\xampp\htdocs\oslonews\resources\views\user.blade.php)' in C:\xampp\htdocs\oslonews\app\Category.php:52
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14628): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#5 C:\xampp\htdocs\oslonews\storage\framework\views\9698c9b52c283cddfbf00a1591f743da(43): Illuminate\View\View->render()
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14626): include('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#9 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#10 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#11 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15623): Illuminate\View\View->render()
#12 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15138): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7688): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8915): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8916): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8901): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\PagesController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'index')
#21 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7855): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'index')
#22 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7826): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7479): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7480): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#29 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7468): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#30 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7453): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2302): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2926): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#37 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12949): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#40 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11536): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#41 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#43 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12686): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#46 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12623): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#48 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#49 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#52 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#53 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#54 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#55 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#56 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#57 {main}

Next exception 'ErrorException' with message 'Trying to get property of non-object (View: C:\xampp\htdocs\oslonews\resources\views\user.blade.php) (View: C:\xampp\htdocs\oslonews\resources\views\user.blade.php)' in C:\xampp\htdocs\oslonews\app\Category.php:52
Stack trace:
#0 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14628): Illuminate\View\Engines\CompilerEngine->handleViewException(Object(ErrorException), 1)
#1 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14662): Illuminate\View\Engines\PhpEngine->evaluatePath('C:\\xampp\\htdocs...', Array)
#2 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14495): Illuminate\View\Engines\CompilerEngine->get('C:\\xampp\\htdocs...', Array)
#3 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14483): Illuminate\View\View->getContents()
#4 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(14474): Illuminate\View\View->renderContents()
#5 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15623): Illuminate\View\View->render()
#6 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(15138): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7688): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8915): Illuminate\Routing\Router->prepareResponse(Object(Illuminate\Http\Request), Object(Illuminate\View\View))
#9 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8916): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(8901): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\PagesController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'index')
#15 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7855): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'index')
#16 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7826): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7479): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#22 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7480): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#23 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7468): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(7453): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2302): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9585): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2926): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#31 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12949): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#34 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(11536): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#37 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12686): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#40 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(12623): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#41 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#43 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2983): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9577): call_user_func_array(Array, Array)
#46 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(9567): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#48 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2249): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#49 C:\xampp\htdocs\oslonews\bootstrap\cache\compiled.php(2232): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#50 C:\xampp\htdocs\oslonews\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#51 {main}  
